packages/api/src/controllers/multistream.ts

```diff
@@ -29,11 +30,17 @@ function adminListQuery(
   orderStr: string,
   filters: string
 ): [SQLStatement[], FindOptions] {
+  type ResultRow = {
```

Comment:
Should the type declaration be outside function?

Reply:
It could be declared externally as well, but since it's a strict local type that never leaves the function I thought it would make sense to keep it local as well, just like a local var (it's allowed by `ts`). I also followed the pattern (that I had started myself 🤣) in the `stream.ts` controller which declares a similar type when calling the generic `db.table.find`.

I don't have a strong opinion about it though and could easily move it outside the function instead. WDYT?

Reply:
resolved in discord conversation